lib/models/os.coffee

```diff
@@ -54,8 +54,10 @@ getOsModel = (deps, opts) ->
 			{ versions, latest } = body
 			[ validVersions, invalidVersions ] = partition(versions, semver.valid)
 
-			validVersions.sort(osVersionRCompare)
```

Comment:
This also highlights that we don't have any tests for this code... If you'd like to add some that'd be wonderful, or you're welcome to just file an issue instead and I'll take a look at it soon.